src/components/info.js

```diff
@@ -1,20 +1,19 @@
-import React, { Component } from 'react'
+import React from 'react'
 import config from '../config'
 import Breadcrumbs from '../partials/breadcrumbs'
 import Tablecontent from '../partials/tablecontent'
 
-export default class Info extends Component {
-  render () {
-    return (
-      <div className='info-wrapper component-wrapper'>
-        <Breadcrumbs title={config.page.info.title} color={config.page.info.color} bgcolor={config.page.info.bgcolor} />
-        <div className='info-content block-content'>
-          <div className='img-wrapper'>
-            <img src={config.page.info.logo} alt='logo' />
-          </div>
-          <Tablecontent info={config.page.info} />
+function Info () {
+  return (
+    <div className='info-wrapper component-wrapper'>
+      <Breadcrumbs title={config.page.info.title} color={config.page.info.color} bgcolor={config.page.info.bgcolor} />
```

Comment:
@phuongdk You should pass `page` instead of `title`, `color`, `bgcolor` as `<Breadcrumbs page={'info'} />`